_gsocproposals/2022/proposal_Belle2Validation.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Advanced Belle II Software Validation
+layout: gsoc_proposal
+project: Belle II
+year: 2022
+organization: LMU
```

Comment:
Project difficulty, duration and mentor availability required (please check the other projects)